doc/nrf/releases/release-notes-1.6.0.rst

```diff
@@ -0,0 +1,471 @@
+.. _ncs_release_notes_160:
+
+Changes in |NCS| v1.6.0
+#######################
+
+.. contents::
+   :local:
+   :depth: 2
+
+|NCS| delivers reference software and supporting libraries for developing low-power wireless applications with Nordic Semiconductor products.
+It includes the TF-M, MCUboot and the Zephyr RTOS open source projects, which are continuously integrated and re-distributed with the SDK.
+
+The |NCS| is where you begin building low-power wireless applications with Nordic Semiconductor nRF52, nRF53, and nRF91 Series devices.
+nRF53 Series devices are now supported for production.
+Wireless protocol stacks and libraries may indicate support for development or support for production for different series of devices based on verification and certification status. See the release notes and documentation for those protocols and libraries for further information.
```

Comment:
- Move the second sentence to a new line.
- If I recall, there were talks about what "production" and "development" actually mean. Not sure there was any consensus, but shouldn't we explain here briefly what these terms mean?

Reply:
Production and development terminology is under discussion but we do not change anything for this release.